packages/pf3-component-mapper/src/tests/form-fields/date-time-picker/date-time-picker.test.js

```diff
@@ -18,5 +18,22 @@ describe('<DateTimePicker />', () => {
     const wrapper = mount(<DateTimePicker />);
     expect(wrapper.state().selectedDay).toBeUndefined();
   });
+
+  it('should initialize date picker value if pristine is set to true', () => {
+    const initialDate = '2019-11-01T12:31:00.000Z';
+    const wrapper = mount(<DateTimePicker value={ initialDate }/>);
+    expect(wrapper.state().selectedDay).toEqual(new Date(initialDate));
+    wrapper.setProps({ pristine: false });
+    wrapper.update();
+
+    wrapper.setProps({ value: '2029-11-01T12:31:00.000Z' });
```

Comment:
Can you set both props at the same time?  Also, I would extract `'2029-11-01T12:31:00.000Z'` to separate const and use it here and in the equal.